iati-organisations-schema.xsd

```diff
@@ -891,5 +781,38 @@
     </xsd:annotation>
   </xsd:attribute>
 
+  <!-- documentLinkWithReceipientCountry extends documentLinkBase to include the receipient-country sub-element, which is used within the iati-organisations schema -->
```

Comment:
I think I did this because I'm not sure if it's possible to extend a `complexType` (i.e. `documentLinkBase`) as an element definition - I think you have to define another `complexType` first. This is also what is suggested by https://stackoverflow.com/a/1684329/2761030

Reply:
👍